packages/adminrouter/buildinfo.json

```diff
@@ -8,9 +8,9 @@
       },
       "adminrouter": {
           "kind": "git",
-          "git": "https://github.com/dcos/adminrouter.git",
-          "ref": "b687a27301a2cc00e11a7f68eccaedf20ce7dd6e",
-          "ref_origin": "master"
+          "git": "https://github.com/vespian/adminrouter.git",
```

Comment:
Added WIP label. To help traceability a bit, PR description could include link to AR PR for now.